ArchiSteamFarm/Utilities.cs

```diff
@@ -183,6 +196,24 @@ public static class Utilities {
 			return (text.Length % 2 == 0) && text.All(Uri.IsHexDigit);
 		}
 
+		[CanBeNull]
+		[PublicAPI]
+		public static IElement SelectSingleNode([NotNull] this IDocument document, string xpath) => (IElement) document.Body.SelectSingleNode(xpath);
+
+		[ItemNotNull]
+		[NotNull]
+		[PublicAPI]
+		public static List<IElement> SelectNodes([NotNull] this IDocument document, string xpath) => document.Body.SelectNodes(xpath).Cast<IElement>().ToList();
```

Comment:
It's worth checking if two same elements (e.g. two same `<div>` clauses) generate the same `IElement` hash-wise and equals-wise. In general, I like to use `HashSet` whenever possible, and this is such a place that needs checking:

If `IElement` is a `class` and two same-value `<div>` generate two standalone objects that are not equal, we should use a `HashSet`.

If `IElement` is a `struct` or two same-value `<div>` generate objects that are equal, we should use a `List` like now, since there might be more nodes with the same values, and we want to preserve them, not omit.

This needs verifying.

Reply:
IElement is interface for a class Element, and hash-wise nodes are not equal, but they are equal-wise.

![image](https://user-images.githubusercontent.com/24438339/77831291-5d467b80-713f-11ea-8294-357ae32044c9.png)

Reply:
List it is then.